external/storm-kafka/README.md

```diff
@@ -120,6 +120,23 @@ spoutConf.scheme = new SchemeAsMultiScheme(new StringScheme());
 OpaqueTridentKafkaSpout spout = new OpaqueTridentKafkaSpout(spoutConf);
 ```
 
+### How KafkaSpout stores offsets of a kafka topic and recovers incase of failures
+
+As shown in the above KafkaConfig properties , user can control where in the topic they can start reading by setting **KafkaConfig.startOffsetTime.**
+
+There are two options **kafka.api.OffsetRequest.EarliestTime()** which makes the KafkaSpout to read from the begining of the topic and 
```

Comment:
I would also document the actual values of `EarliestTime()` (`-2`) and `LatestTime()` (`-1`), and that it can also be set to a point in time (a la `System.currentTimeMillis()`).

My reasoning behind documenting the values (as opposed to the kafka API constants) is that the start offset time is likely to be specified via configuration (i.e. outside java code). Either that, or add spout constants that would get evaluated to `EarliestTime()`/`LatestTime()` if for some reason those values were ever changed in the Kafka API -- that seems like a less "leaky" solution.